Pear/View/OnboardingSearchDropdownView.swift

```diff
@@ -124,6 +142,23 @@ class OnboardingSearchDropdownView: UIView {
         delegate?.updateSelectedFields(tag: tag, isSelected: true, valueSelected: title)
     }
 
+    func searchPlaces(place: String) {
+        debounceTimer?.invalidate()
```

Comment:
what do we need a debouncer for? are we limited on the requests we can make to the API or is it to be on the safe side if anyone tries anything malicious?

Reply:
we dont wanna make too many requests at once so this throttles the requests